macros/GroupData.json

```diff
@@ -1834,10 +1857,15 @@
                             "RTCSessionDescriptionCallback",
                             "RTCStatsReport",
                             "RTCCertificate",
+                            "RTCIdentityProvider",
+                            "RTCIdentityAssertion",
+                            "RTCIdentityEvent",
+                            "RTCIdentityErrorEvent",
                             "RTCIdentityProviderRegistrar",
+                            "RTCTrackEvent",
                             "RTCRtpReceiver",
                             "RTCRtpSender" ],
-            "methods":    [],
+            "methods":    [ "Navigator.mediaDevices.getUserMedia()"],
```

Comment:
This is how you would call the method in code. The docs live under "MediaDevices.getUserMedia()" which we want to link to here.